osu.Game/Graphics/Containers/ScalingContainer.cs

```diff
@@ -36,6 +36,24 @@ public class ScalingContainer : Container
 
         private BackgroundScreenStack backgroundStack;
 
+        private bool allowScaling = true;
+
+        /// <summary>
+        /// Whether user scaling preferences should be applied. Enabled by default.
+        /// </summary>
+        public bool AllowScaling
+        {
+            get => allowScaling;
+            set
+            {
+                if (value == allowScaling)
+                    return;
+
+                allowScaling = value;
+                updateSize();
```

Comment:
Bit of a side note, but this isn't too safe. `updateSize()` adds transforms, so if anyone changes this value off of the update thread this'll crash.

Reply:
Have added a guard against this one.